bin/run-dev

```diff
@@ -1,57 +1,61 @@
-#! /bin/bash
+#! /usr/bin/env bash
 
 # DOC: Run the app locally using Docker. Useful for manual testing.
 
-pushd $(git rev-parse --show-toplevel) > /dev/null
-
-set -e
-
-bin/pull-image
-
-if [ -f ".secrets" ]; then
-    . .secrets
-fi
-
-
-# Get cloud provider (currently either AWS or Azure)
-_setArgs(){
+source bin/lib.sh
+
+# Default to using Localstack emulator.
+emulators::set_localstack_emulator_vars
+
+#######################################
+# Process the script's commandline args.
+# Globals:
+#   already_set_cloud_provider
+#   cloud_provider
+#   emulator
+#   server_url
+#   STORAGE_SERVICE_NAME
+# Arguments:
+#   1: "$@" - full args array for the script
+#######################################
+function set_args() {
   while [ "${1:-}" != "" ]; do
     case "$1" in 
       "--azure")
-       cloudProvider="azure"
-       emulator="azurite"
-       export STORAGE_SERVICE_NAME="azure-blob"
-       ;;
+        emulators::ensure_only_one_cloud_provider_flag azure
+        emulators::set_azurite_emulator_vars
+        ;;
+
       "--aws")
-       cloudProvider="aws"
-       emulator="localstack"
-       export STORAGE_SERVICE_NAME="s3"
-       ;;
+        emulators::ensure_only_one_cloud_provider_flag aws
+        # Already defaulted to AWS.
+        ;;
     esac
+
     shift
   done
 }
 
-_setArgs $*
+set_args "$@"
+
+echo "Making sure we're up to date with the latest dev... "\
+  "set environment variable USE_LOCAL_CIVIFORM=1 to skip"
```

Comment:
likewise needing export.

However... this is redundant with the echo in pull-image, but we hide the scripts operation.... Maybe we don't want to?

Reply:
there's a bunch of docker messages that print out that really cluttered the output, imo suppressing that made scanning the output easier and of higher value